### DISCUSSION TITLE
被拒绝注册

### DISCUSSION THREAD

**Discussion by everfu** [zeabur/.github#19] [General]
您好，当我想要用GitHub注册并登录的时候，显示“You are not allwed to signup”
![3ec98cf6c49ba02be9b4ef9fbbc6dcf1](https://github.com/zeabur/.github/assets/74389842/71ac9db6-736a-4cd1-8b3f-e3f63ab6e4ab)

**Comment by scmtble** [comment]
您好，可以提供一下邮箱吗，现在对一些匿名邮箱有限制

**Reply by kilingzhang** [reply]
You need to agree to the service terms .

icoud的匿名邮箱也会触发风控吗，并且提示信息很不友好。

**Reply by everfu** [reply]
a2230438902@outlook.com
